packages/@aws-cdk/aws-stepfunctions-tasks/test/integ.glue-task.ts

```diff
@@ -0,0 +1,55 @@
+import * as glue from '@aws-cdk/aws-glue';
+import * as iam from '@aws-cdk/aws-iam';
+import * as assets from '@aws-cdk/aws-s3-assets';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as path from 'path';
+import * as tasks from '../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn>
```

Comment:
Capture the state machine ARN as a `CfnOutput` so it's printed clearly in the console output.

This doesn't verify much. At a minimum, there must be a 2nd step that the job succeeded.

Can we also add steps to check that the glue job run succeeded by calling `GetJobRun`?